addons/web/static/src/views/fields/properties/property_value.xml

```diff
@@ -92,6 +92,7 @@
                     </t>
```

Comment:
From commit message: is it only an issue with the "search more" feature?

When you type 3 characters into the box, does it search on the correct model then?